seed: 42
device: cuda:0
num_steps_per_env: 24
max_iterations: 15000
empirical_normalization: false
obs_groups:
  policy:
  - policy
  critic:
  - critic
clip_actions: null
save_interval: 250
experiment_name: ymboy_skate_rnn
run_name: ''
logger: tensorboard
neptune_project: isaaclab
wandb_project: isaaclab
resume: true
load_run: .*
load_checkpoint: model_.*.pt
class_name: OnPolicyRunner
policy:
  class_name: ActorCriticRecurrent
  init_noise_std: 1.0
  noise_std_type: scalar
  actor_obs_normalization: {}
  critic_obs_normalization: {}
  actor_hidden_dims:
  - 512
  - 256
  - 128
  critic_hidden_dims:
  - 512
  - 256
  - 128
  activation: elu
  rnn_hidden_size: 256
  rnn_num_layers: 1
  rnn_type: lstm
algorithm:
  class_name: PPO
  num_learning_epochs: 5
  num_mini_batches: 4
  learning_rate: 0.001
  schedule: adaptive
  gamma: 0.99
  lam: 0.95
  entropy_coef: 0.01
  desired_kl: 0.01
  max_grad_norm: 1.0
  value_loss_coef: 1.0
  use_clipped_value_loss: true
  clip_param: 0.2
  normalize_advantage_per_mini_batch: false
  rnd_cfg: null
  symmetry_cfg: null
